refactor.log

```diff
@@ -0,0 +1,8 @@
+======================   Refactor log   ======================
+
+Protocol of the changes applied to the executable 'metamath'.  Each change
+should correspond to a small set of commits implementing this change, and
+nothing else (apart from cosmetic corrections).
+
+[wl c6a76d9] Comment update: Better explain the design principles of the fatal
+   error handler in the 'Rationale' section.
```

Comment:
Why are you introducing yet another way to track changes? There is already a changelog in metamath.c.

Reply:
This is temporary, focusing on my work, and more fine grained.  Since refactoring is about small steps, the change log would grow extremely, even protocolling things that get reversed shortly after.  Do you want to see an entry for this commit for example? If you wish, I will remove it again, but that will take effect only after the currently pending commits are done.

Reply:
I would prefer you put those kind of fine-grained comments in the commit messages themselves, or the PR description. The code itself should not have temporary stuff in it, leave that in your own local copy.